### ISSUE TITLE
Integers in YAML config results into error

### ISSUE THREAD

**Issue by vjuranek** [vjuranek/radargun-plugin#45]
Setting up property to an integer value in YAML config (e.g. `aaa = 1`) resutls into an error. It accepts only strings.

**Comment by vjuranek** [comment]
'''[RadarGun] ERROR: somethign went wrong, caught exception java.lang.Integer cannot be cast to java.lang.String'''

**Comment by vjuranek** [comment]
Full exception:
```
java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.String
	at org.jenkinsci.plugins.radargun.config.ScriptSource.prepareEnvVars(ScriptSource.java:97)
	at org.jenkinsci.plugins.radargun.config.ScriptSource.getNodeCmdLine(ScriptSource.java:60)
	at org.jenkinsci.plugins.radargun.config.ScriptSource.getMasterCmdLine(ScriptSource.java:76)
	at org.jenkinsci.plugins.radargun.model.impl.RgMasterProcessImpl.getMasterCmdLine(RgMasterProcessImpl.java:67)
	at org.jenkinsci.plugins.radargun.model.impl.RgMasterProcessImpl.createRunner(RgMasterProcessImpl.java:38)
	at org.jenkinsci.plugins.radargun.model.impl.AbstractRgProcess.start(AbstractRgProcess.java:21)
	at org.jenkinsci.plugins.radargun.RadarGunBuilder.perform(RadarGunBuilder.java:145)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:770)
	at hudson.model.Build$BuildExecution.build(Build.java:199)
	at hudson.model.Build$BuildExecution.doRun(Build.java:160)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:533)
	at hudson.model.Run.execute(Run.java:1745)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:89)
	at hudson.model.Executor.run(Executor.java:240)
```

**Comment by vjuranek** [comment]
Fixed in 7df050a89d7605ceccca6d7d574ef51474e05519
